src/utils/react-modals.js

```diff
@@ -33,21 +59,30 @@ function createModalContextValue (
 export const WebAdminModalContext = React.createContext(createModalContextValue())
 WebAdminModalContext.displayName = 'WebAdminModalContext'
 
+/**
+ * Create an application container `div` in the wedadmin window and render the
+ * given `render()` as a React app in the container.  A `WebAdminModalContext.Provider`
+ * configured with the correct `targetWindow` and `targetContianer` as determine and
+ * created by the function.
+ *
+ * @param {function} render Function to render your React App root
+ * @param {string} id element id for the container div
+ */
 export const renderComponent = (render, id) => {
   const targetWindow = getWebAdminWindow()
 
   let container = targetWindow.document.querySelector(`div#${id}`)
   if (!container) {
     container = targetWindow.document.createElement('div')
     container.setAttribute('id', id)
-    container.setAttribute('class', 'ui-extensions-container')
+    container.setAttribute('class', 'ui-extensions-plugin-approot')
```

Comment:
please comment that this is only a marker class

Reply:
Done.